README.md

```diff
@@ -539,6 +544,109 @@ spec:
 
 **Note:** The `workbenchNamespace` field once set, it cannot be changed (immutable).
 
+### Example GatewayConfig
+
+The GatewayConfig custom resource is used to configure gateway settings for OpenDataHub, including OIDC authentication and ingress gateway configuration. This CR is cluster-scoped and must be named `default-gateway`.
+
+**Automatic Creation**: The GatewayConfig CR is automatically created when a DSCInitialization CR is applied to the cluster. Users typically don't need to create this CR manually unless they want to configure OIDC authentication mode or customize ingress gateway settings.
+
+Here's an example of the default GatewayConfig CR (automatically created):
+
+```yaml
+apiVersion: services.platform.opendatahub.io/v1alpha1
+kind: GatewayConfig
+metadata:
+  name: default-gateway
+spec:
+  cookie: {}  # Uses defaults: expire: 24h, refresh: 1h
+  certificate:
+    type: OpenshiftDefaultIngress
+```
+
+For a setup with custom certificate (Provided type):
+
+```yaml
+apiVersion: services.platform.opendatahub.io/v1alpha1
+kind: GatewayConfig
+metadata:
+  name: default-gateway
+spec:
+  cookie: {}  # Uses defaults: expire: 24h, refresh: 1h
+  domain: "*.apps.example.com"
+  certificate:
+    type: Provided
+    secretName: custom-tls-cert  # Secret must already exist in openshift-ingress namespace with tls.crt and tls.key
+```
+
+For an advanced example with OIDC authentication:
+
+```yaml
+apiVersion: services.platform.opendatahub.io/v1alpha1
+kind: GatewayConfig
+metadata:
+  name: default-gateway
+spec:
+  oidc:
+    issuerURL: "https://keycloak.example.com/auth/realms/opendatahub"
+    clientID: "opendatahub-client"
+    clientSecretRef:
+      name: oidc-client-secret
+      key: client-secret
+    secretNamespace: "my-custom-namespace"
```

Comment:
I don’t think making the secret namespace configurable is a good idea. It could create security risks, as an unauthorized user might be able to create a fake client and client secret.

Restricting secret to the OpenShift-Ingress namespace has allowed only the admin user to add /update the secret